### ISSUE TITLE
Stop asking me on every file move if its a copy, move, or create symlink etc. Allow the user to pick a default to skip asking on every drag and drop.

### ISSUE THREAD

**Issue by thrisp** [lxqt/pcmanfm-qt#993]
What the title says.

**Comment by tsujan** [comment]
Already mentioned and rejected.

**Comment by thrisp** [comment]
ahhh why, link please.

**Comment by tsujan** [comment]
> ahhh why, link please.

I don't have time to find it but have time for a short explanation, whose link could be used later.

**Regarding all such feature requests:**

A user may want iconless menus, another may not want a statusbar, yet another may dislike item tooltips, etc. Even if adding options for them made sense, it would result in cluttered preferences dialogs like those of KDE.

**Specifically:**

Limiting DND to copying, moving or link creation is too personal. A user may find an extra click annoying, intolerable or agonizing but there's nothing I can do.

**In short:**

Unlimited customization is not an option for an app. Whether some developers do it or not is irrelevant.

**Comment by 1xch** [comment]
Thank you for explaining the logic of this to me.
